### PR TITLE
IA-1112: Adding kinesis firehose for WAF logs

### PR DESCRIPTION
**Pull request by bushong1**

### CODE REVIEW

File: terraform/account/modules/waf-global/kinesis-firehose-logs.tf [hunk 1]
```diff
@@ -0,0 +1,32 @@
+resource "aws_iam_role" "firehose_role" {
+  count = var.kinesis_firehose_logs_enabled ? 1 : 0
+  name  = "${var.waf_prefix}_waf_firehose_role"
```
Comment (jsmigovsky) [inline terraform/account/modules/waf-global/kinesis-firehose-logs.tf]: This appears the be a duplicate name.

File: terraform/account/modules/waf-regional/kinesis-firehose-logs.tf [hunk 2]
```diff
@@ -0,0 +1,32 @@
+resource "aws_iam_role" "firehose_role" {
+  count = var.kinesis_firehose_logs_enabled ? 1 : 0
+  name  = "${var.waf_prefix}_waf_firehose_role"
```
Comment (jsmigovsky) [inline terraform/account/modules/waf-regional/kinesis-firehose-logs.tf]: This appears to be a duplicate name.

File: terraform/account/modules/waf-global/kinesis-firehose-logs.tf (hunk 1 above)
Comment (bushong1) [reply terraform/account/modules/waf-global/kinesis-firehose-logs.tf]: `waf_prefix` is a variable made to avoid the same name problem

Comment (jsmigovsky) [reply terraform/account/modules/waf-global/kinesis-firehose-logs.tf]: gotcha

**Comment by bushong1** [comment]
issues with IAM roles; not gonna merge it

**Pull request merged**